baggers.js

```diff
@@ -14661,7 +14730,7 @@ var data = {
     },
     Caen: {
       name: 'Caen',
-      ville_img: 'img/villes/BBL_Caen.png',
+      ville_img: 'img/villes/BBL_Caen.jpg',
```

Comment:
Even though it's not related to your PR I think this should be: `images/villes/caen.jpg` instead.

Reply:
Ok, it's done
(But all other links to a city's picture are with the form `img/cities/BBL_cityName.imageType`)

Reply:
Hmmm. My bad... Please re-revert what I asked for. Although I think we don't use that image anymore.

Instead we should change http://www.brownbaglunch.fr/css/baggers.css file and add:

```css
.bagger-header.caen { background: url(../images/villes/caen.jpg) no-repeat center center; }
```